tests/test_dsp.py

```diff
@@ -111,6 +111,18 @@ def test_group_delay_custom_frequencies(impulse_group_delay):
         grp, impulse_group_delay[1][0, frequency_idx], atol=1e-10)
 
 
+@pytest.mark.parametrize("shape", [(4, 1), (1, 4)])
```

Comment:
```suggestion
@pytest.mark.parametrize("shape", [(4, 1), (1, 4), (1, ), (1, 1)])
```
i think input (1,) should also return (1,..). maybe we need to discuss. in the doc its mentioned : "The array is flattened if a single channel signal was passed to the function."

Reply:
Yeah, i remember that we talked about keeping the input shape in any case for all the dsp functions.
I am on it!